src/test/java/core/basesyntax/service/impl/ConverterImpTest.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.ItemTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.Converter;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class ConverterImpTest {
+    private final Converter converter = new ConverterImpl();
+
+    @Test
+    void convert_ok() {
+        List<String> input = new ArrayList<>();
+        input.add("type,fruit,quantity");
+        input.add("b,banana,20");
+        input.add("b,apple,100");
+        input.add("s,banana,100");
+        input.add("p,banana,13");
+        input.add("r,apple,10");
+        input.add("p,apple,20");
+        input.add("p,banana,5");
+        input.add("s,banana,50");
+        List<ItemTransaction> expected = new ArrayList<>();
+        expected.add(new ItemTransaction(Operation.BALANCE,"banana", 20));
+        expected.add(new ItemTransaction(Operation.BALANCE,"apple", 100));
+        expected.add(new ItemTransaction(Operation.SUPPLY,"banana", 100));
+        expected.add(new ItemTransaction(Operation.PURCHASE,"banana", 13));
+        expected.add(new ItemTransaction(Operation.RETURN,"apple", 10));
+        expected.add(new ItemTransaction(Operation.PURCHASE,"apple", 20));
+        expected.add(new ItemTransaction(Operation.PURCHASE,"banana", 5));
+        expected.add(new ItemTransaction(Operation.SUPPLY,"banana", 50));
+        List<ItemTransaction> actual = converter.convert(input);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void convert_invalidData() {
```

Comment:
invalid test method naming
```suggestion
    void convert_invalidData_notOk() {
```